### DISCUSSION TITLE
How many devices can I use Rectangle Pro on? What happens if a reinstall macOS?

### DISCUSSION THREAD

**Discussion by rxhanson** [rxhanson/RectanglePro-Community#42] [Q&A] [answered]
How many devices can I use Rectangle Pro on?

What happens if I reinstall macOS?

**Comment by rxhanson** [comment, marked as answer]
> How many devices can I use Rectangle Pro on?

Each license can be activated on 3 Macs at once.

You can deactivate a license from the Settings tab of the preferences window, or from the About window. If you wish to transfer a license activation to a new Mac.

> What happens if I reinstall macOS?

Your license activation is tied to your Mac, ~~and unfortunately there is no way to remove an activation from another device. This means that you will need to deactivate your license from within the app before you reinstall macOS or get rid of your Mac. This is a limitation from Paddle, the company that handles the licenses and fulfills the purchases for Rectangle Pro. Sorry for the inconvenience.~~

Update: If you have a new Mac or reinstall macOS and now have no way to remove an activation on your old Mac or previous install, you can now remove your oldest activation from within Rectangle Pro on your new Mac/install. **In the settings tab of the preferences window, there is a button for removing the oldest activation.**

**Reply by ziptbm** [reply]
> 
> > What happens if I reinstall macOS?
> 
> Your license activation is tied to your Mac, and unfortunately there is no way to remove an activation from another device. This means that you will need to deactivate your license from within the app before you reinstall macOS or get rid of your Mac. This is a limitation from Paddle, the company that handles the licenses and fulfills the purchases for Rectangle Pro. Sorry for the inconvenience.

Ok, I'm seeing this now as I have Rectangle Pro active on 3 Macs.  I had one of them completely die on me and Apple replaced it.  However, I was unable to deactivate the license before that device crashed and was subsequently replaced.  I'm now trying to activate on the replacement and I am unable to.  What options do Ihave to get around this?

**Reply by rxhanson** [reply]
Send me an email with either your license key or the email address used to purchase the license key and I’ll get it sorted out.
